### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_ImportBacpac_ImportBacpacSettings_ScreenReader:The selected file's location is not announced. 

### ISSUE THREAD

**Issue by fsteffi** [nvaccess/nvda#10365]
**GitHubTags:**#A11yMAS; #A11yTCS; #A11y_ADS_Schema_Dacpac_Backup; #Azure Data Studio; #DesktopApp; #Win10; #NVDA; #Jaws; #MAS1.3,1; #MAS4.2.12;

**Environment Details:**
Jaws version:2019.1907.42.ILM
NVDA version:2019.1.1

**Additional Details:**
Tools: Jaws, NVDA
MAS References:MAS1.3.1, MAS4.2.12

**Repro Steps:**
1.Launch Azure Data Studio application.
2.Connect to server with windows credentials.
3.Navigate to Extensions from left menu.
4.Search & install Dacpac extension.
5.Navigate to Connections from left menu.
6.Expand the server connected in step 2.
7.Right click an on prem database and select Data-tier application wizard from Context menu.
8.Select third radio button.
9.Navigate till next button & hit enter.
10.Start screen reader as nvda.
11.Navigate using tab key to the "..." button file location and select a file.

**Actual:**
On selecting a file, the newly added file location is not announced by the screen reader , the announcement is just the dialog and the button name.

Similar issue is observed in jaws configuration.

**Expected:**
The file location of the new file added should be announced as soon as it has been added in the edit box.

**User Impact:**
Screen reader users will not know the location of the newly added file as soon as it is added.

[#6795A11y_AzureDataStudio(Schema,Dacpac,Backup)_ImportBacpac_ImportBacpacSettings_ScreenReader.pptx](https://github.com/nvaccess/nvda/files/3711089/6795A11y_AzureDataStudio.Schema.Dacpac.Backup._ImportBacpac_ImportBacpacSettings_ScreenReader.pptx)

**Comment by XLTechie** [comment]
In general, you should be testing with the most recent public NVDA: 2019.2.

Also, does this problem occur in latest NVDA Alpha? 
https://www.nvaccess.org/files/nvda/snapshots/

**Comment by fsteffi** [comment]
@XLTechie the page was tested with nvda 2019.2

**Comment by michaelDCurran** [comment]
Closing on request from reporter privately by email.
